lib/Dialect/CKKS/Conversions/CKKSToLWE/CKKSToLWE.td

```diff
@@ -0,0 +1,20 @@
+#ifndef LIB_DIALECT_CKKS_CONVERSIONS_CKKSTOLWE_CKKSTOLWE_TD_
+#define LIB_DIALECT_CKKS_CONVERSIONS_CKKSTOLWE_CKKSTOLWE_TD_
+
+include "mlir/Pass/PassBase.td"
+
+def CKKSToLWE : Pass<"ckks-to-lwe"> {
+  let summary = "Lower `ckks` to `lwe` dialect.";
+
+  let description = [{
+    This pass lowers the `ckks` dialect to `lwe` dialect.
```

Comment:
here and for BGV, please mention that some scheme-specific ops that have no analogue in `lwe` are not lowered, and must be handled directly by the later pass that lowers to an exit dialect.